src/others/components/map/CircleLayerStyle.tsx

```diff
@@ -6,6 +6,7 @@ export const layerStyle: LayerProps = {
   minzoom: 2,
```

Comment:
**suggestion**

`.tsx` extension suggests it's a React component which it's not. Consider changing it to `.ts`

Reply:
done